### PR TITLE
Bugfix/materials build broken

### LINKED ISSUE #2202: build error when trying to build with the latest version of materials
in our esales-ui-component-library, we wanted to import the latest version of the axa materials, cause patterns library replaced node-sass with sass in that latest version.
when we imported the latest materials version and try to build, we got the following error message:
image

so at the moment we are not able to use the latest version of axa-materials.

thanks for your help.

best regards,

Quan

### PR DESCRIPTION
**Pull request by 01231**
Fixes #2202

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.
- [x] Dependencies to other components still work.

### CODE REVIEW

File: src/components/00-materials/CHANGELOG.md
Comment (markus-walther) [inline src/components/00-materials/CHANGELOG.md]: Better: Revert to the state of ... not possible for **us** to use...

File: package.json
Comment (markus-walther) [inline package.json]: why this change? Just explain here.

**Review by markus-walther — CHANGES_REQUESTED** [review]
Looks good in general, just some small changes from my point of view.

And a question: is this a formal, automatic revert, or did you revert manually (by typing the replacement lines)? I.e. how much should we trust the revert is correct?

File: package.json
Comment (01231) [reply package.json]: It was node-sass befor I tried to fix #2177. I updated to sass because then I would be able to use the @rules. So I'm reverting this also to the state from around 3 weeks.

File: src/components/00-materials/CHANGELOG.md
Comment (01231) [reply src/components/00-materials/CHANGELOG.md]: done

**Comment by 01231** [comment]
@markus-walther Yes I did it manually. But because I compared the changes side by side and I'm the author of the previous PR I'm pretty confident that I haven't missed a thing.

**Comment by marek-polak** [comment]
Hi, any update on this one? We would like to be able to build our esales-components again :)

**Comment by 01231** [comment]
@marek-polak Hi Marek, the PR is waiting for an approval. This won't be earlier than next monday because of various abcences in our team.

File: package.json
Comment (markus-walther) [reply package.json]: OK, makes sense

**Review by markus-walther — APPROVED** [review]
LGTM

**Pull request merged**
